docs/PRE-COMMIT.md

```diff
@@ -10,3 +10,13 @@ The pre-commit hook in this repository uses various plugins to run different kin
 1. [Pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks): Checks and fixes basic linting issues. All available hooks for pre-commit can be found [here](https://github.com/pre-commit/pre-commit-hooks#hooks-available).
 
 2. [Prettier mirror](https://github.com/pre-commit/mirrors-prettier): Checks for code formatting issues within Javascript files and fixes if errors found, like indentation, extra lines and spaces, invalid syntax, etc .
+
+#Installation
+Install pre-commit package manager.
+    pip install pre-commit
+
+You can install it in your codespace using this: https://github.com/avantifellows/plio-frontend/blob/master/docs/PRE-COMMIT.md.
+
+#Usage
+Run command:
+    pre-commit .
```

Comment:
please look at `docs/INSTALLATION.md` for how to correctly set headings (this has no space between `#` and the text) - did you actually see how it looks in the markdown file?